.gitignore

```diff
@@ -0,0 +1,8 @@
+# Ignore node_modules folder
+node_modules/
+
+# Ignore .github folder
+.github
+
+# Ignore .github folder
+test.md
```

Comment:
- [ ] You did a great job 😃 at adding the test file and node modules folder to the gitignore file, but I see that you put the .github folder which of the configuration of the linter in the gitignore as well, so kindly : 
- Try 💯 to remove the .github folder from the gitignore to let the linter works properly as it is not practice to put the linter inside the gitignore as we want to see that the linter is working in GitHub  😊